src/Onboarding/Welcome.tsx

```diff
@@ -44,50 +50,60 @@ const Welcome: FunctionComponent = () => {
   return (
     <>
       <StatusBar backgroundColor={Colors.primaryLightBackground} />
-      <GradientBackground
-        gradient={Colors.gradientPrimary10}
-        angleCenterY={0.75}
+      <ScrollView
+        alwaysBounceVertical={false}
+        contentContainerStyle={style.scrollContentContainer}
```

Comment:
We tend to call this `style.contentContainer` elsewhere in the app but totally a matter of preference.